resources/views/components/action.blade.php

```diff
@@ -1,9 +1,32 @@
 {{-- This renderIf method is implemented in every action --}}
 @php
-  $class = 'flex hover:text-blue-600 transition-all duration-300 ease-in-out focus:text-blue-600 active:text-blue-600"'
+    $class = 'flex hover:text-blue-600 transition-all duration-300 ease-in-out focus:text-blue-600 active:text-blue-600"'
 @endphp
 @if ($action->renderIf($item))
-  <a href="#!" title="{{$action->title}}" wire:click.prevent="executeAction('{{ $action->id }}', '{{ $item->getKey() }}', true)" class="{{ $class }}">
-    {{ $slot }}
-  </a>
+
+    <div x-data="{ tooltip: false }" class="cursor-pointer relative z-30 inline-flex"
+         @mousemove.away="tooltip = false"
+    >
+        <button
```

Comment:
Why a button?

Reply:
Because I see it was more a button than a link, as it is not linking to anywhere